onadata/libs/utils/presigned_download_url.py

```diff
@@ -0,0 +1,44 @@
+import boto3
+from botocore.exceptions import ClientError
+import logging
+
+from django.conf import settings
+from django.core.files.storage import get_storage_class
+from django.http import HttpResponse, HttpResponseRedirect
+
+from wsgiref.util import FileWrapper
+
+
+def generate_media_download_url(file_path: str, expiration: int = 3600):
+    default_storage = get_storage_class()()
+    filename = file_path.split("/")[-1]
+    s3 = get_storage_class('storages.backends.s3boto3.S3Boto3Storage')()
+
+    if default_storage.__class__ != s3.__class__:
+        file_obj = open(settings.MEDIA_ROOT + file_path, 'rb')
+        response = HttpResponse(FileWrapper(file_obj),
+                                content_type='image/jpg')
```

Comment:
Are we expecting the content type to always be `image/jpg`?

Reply:
Makes sense, I will change that